examples/simple_example/README.md

```diff
@@ -0,0 +1,29 @@
+# Simple Example
```

Comment:
In terms of examples, we discussed that there are 3 main uses for this module - one to use both the managed zone and record set, one to manage managed zone only, and one to manage the record set only. Having these three examples will be useful as test cases as well, since we're exercising more execution paths.